src/assets/css/screens/gameScreen.css

```diff
@@ -1,5 +1,6 @@
 .game__screen {
   background-repeat: no-repeat;
+  background-size: cover;
```

Comment:
I removed it because there was a problem with it on small height

Reply:
check game page, when dwarves class in selected